src/test/java/net/datafaker/providers/base/TwitterTest.java

```diff
@@ -91,29 +92,36 @@ void testTextKeyWords() {
 
     @Test
     void username() {
+        final Twitter twitter = faker.twitter();
```

Comment:
Does this need to be instantiated for each method or should it just be a class member?

Reply:
yes, makes sense to extract it